internal/ssh/handler.go

```diff
@@ -264,7 +264,12 @@ func (h *SessionHandler) SignFile(sesh *UserSession, file *snips.File) {
 }
 
 func (h *SessionHandler) DownloadFile(sesh *UserSession, file *snips.File) {
-	wish.Print(sesh, string(file.Content))
+	content, err := file.GetContent()
+	if err != nil {
+		sesh.Error(err, "Unable to download file", "There was an error downloading the file: %q", file.ID)
+	} else {
+		wish.Print(sesh, content)
```

Comment:
```suggestion
		wish.Print(sesh, string(content))
```

found this when testing locally 😉 it was printing out the byte slice haha

Reply:
sorry! was caught up in the unit tests :(